packages/api/src/router/testSupport/seedChat/index.ts

```diff
@@ -0,0 +1,31 @@
+import { prisma } from "@oakai/db";
+
+import typicalChatJson from "./typical.json";
+
+const fixtures = {
+  typical: typicalChatJson,
+};
+
+export const seedChat = async (
+  userId: string,
+  fixtureName: keyof typeof fixtures,
+) => {
+  const id = `e2e-typical-user${userId}`;
```

Comment:
Should this have a dash before userId?

Reply:
I just had a look in my DB and this is actually generating IDs like `e2e-typical-useruser_2mZJKBdphP9dckutEjfwzcfmYei`. So I'm going to remove this prefix anyway. Thanks for the prompt to have a look